code/__DEFINES/access.dm

```diff
@@ -86,6 +86,8 @@
 #define ACCESS_FREEMINER 71
 #define ACCESS_FREEMINER_CAPTAIN 72
 #define ACCESS_FREEMINER_ENGINEER 73
+//Yogs: Brig Psych access, used for their locker and offices if one is added
```

Comment:
nitpicky but we already have a Psych so this might be confusing for people that don't know about the Brig Physician

Reply:
I misspelled it, its phys not psych